gobblin-modules/gobblin-eventhub/src/main/java/gobblin/eventhub/writer/EventhubDataWriterBuilder.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package gobblin.eventhub.writer;
+
+import java.io.IOException;
+
+
+import gobblin.writer.DataWriter;
+import gobblin.writer.DataWriterBuilder;
+
+
+/**
+ * Builder that hands back a {@link EventhubDataWriter}
+ */
+public class EventhubDataWriterBuilder extends DataWriterBuilder {
+
+    @Override
+    public DataWriter build()
+        throws IOException {
+
+      return null;
```

Comment:
This does not look write. I assume it should be

    return new EventhubDataWriter ...